rmgpy/tools/fluxtest.py

```diff
@@ -0,0 +1,24 @@
+import unittest
+import os
+import os.path
+import shutil
+
+from rmgpy.tools.fluxdiagram import *
+
+class FluxDiagramTest(unittest.TestCase):
+
+    def test_avi(self):
+        folder = os.path.join(os.getcwd(),'rmgpy/tools/data/flux/')
```

Comment:
Good suggestion. I don't actually use Windows so can't test this, and it may make no difference, but I am in the habit of writing it like this (because Windows uses backslashes in paths instead of slashes?):
```python
folder = os.path.join(os.path.dirname(rmgpy.__file__),'tools','data','flux')
```

(if it does make a difference, may want to change lines 16 and 17 below)